src/controllers/controller.bar.js

```diff
@@ -87,7 +87,14 @@ defaults._set('horizontalBar', {
 
 			label: function(item, data) {
 				var datasetLabel = data.datasets[item.datasetIndex].label || '';
-				return datasetLabel + ': ' + item.xLabel;
+				//  float-bar support, if y arguments are array tooltip will show bottom and up values
+				var yValue = data.datasets[item.datasetIndex].data[item.index];
+				if (helpers.isArray(yValue)) {
```

Comment:
i agree with the last sugegstion - its done.

as for getLabelForIndex()

`getLabelForIndex: function(index, datasetIndex) {
			return +this.getRightValue(this.chart.data.datasets[datasetIndex].data[index]);
		},`

It comes from there and just return the label that shouldnt be changed when scaling:

`xLabel: xScale ? xScale.getLabelForIndex(index, datasetIndex) : '',`

also, i think we can omit passing values for labels to getRightValue(), cause this values were already checked when bar is drawn. So, if we have some problems with values it will affect firstly the process of bar drawing. What do you think?